cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -77,15 +77,15 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Check dependencies
 
     private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
 
-    private val providerFactory by option(
-        "--advisor", "-a",
-        help = "The advisor to use, one of ${allVulnerabilityProvidersByName.keys}"
+    private val providerFactories by option(
+        "--vulnerability-providers", "-v",
+        help = "The comma separated providers to use, any of ${allVulnerabilityProvidersByName.keys}"
     ).convert { name ->
         allVulnerabilityProvidersByName[name]
             ?: throw BadParameterValue(
-                "Advisor '$name' is not one of ${allVulnerabilityProvidersByName.keys}"
+                "Provider '$name' is not one of ${allVulnerabilityProvidersByName.keys}"
             )
-    }.required()
+    }.split("").required()
```

Comment:
I think this should be `.split(',')`, shouldn't it?

Reply:
In this case it does not seem to work with single quotes, but you are right about the comma.